includes/core/class-loader.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Core loader.
+ *
+ * @package WooCommerce\Payments
+ */
+
+/** Load contracts. */
+
+require_once __DIR__ . '/Contracts/API/Request/interface-base-request.php';
+require_once __DIR__ . '/Contracts/API/Response/interface-base-response.php';
+
+/** Load DTOs. */
+
+require_once __DIR__ . '/DataTransferObjects/class-data-transfer-object.php';
+require_once __DIR__ . '/DataTransferObjects/API/class-response.php';
+
+/** Load enums. */
+
+require_once __DIR__ . '/Enums/class-http-methods.php';
+require_once __DIR__ . '/Enums/class-wc-pay-endpoints.php';
+
+/** Load value objects */
+
+require_once __DIR__ . '/ValueObjects/class-value-object.php';
+require_once __DIR__ . '/ValueObjects/API/Request/class-base-request.php';
+require_once __DIR__ . '/ValueObjects/API/Request/class-create-charge.php';
```

Comment:
The problem with autoloader is that it was called for every loaded class after it's called, and if statement with namespace check had to be added to prevent any issues with loading. With all that in mind, I remove it to prevent any issues with performance. I haven't faced any performance issues, but I wasn't 100% certain, and that's why I removed it.

Reply:
We chatted about this: Let's convert the core files, and add autoloading when the time comes. In the meantime, let's:

1. Make the directory names follow WPCS (lowercase in this case).
2. Include the files instead of having a `class-autoloader.php`, which doesn't autoload anything.

Reply:
Maybe we can autoload/preload files with `spl_autoload_register` like I did in commits before, that will support preloading of core directory by using namespace to find a path to file. What do you think?

Reply:
All folders have been replaced with WPCS standard from the last commit and autoloader class have been deleted and all requires have been placed in `class-wc-payments.php` file where majority of classes that are required are presented.